### ISSUE TITLE
Container images still point to rgardler/simdem*

### ISSUE THREAD

**Issue by lastcoolnameleft** [Azure/simdem#6]
Since this is in Azure Org + CircleCI, this should probably be updated to the Azure namespace.

**Comment by SorraTheOrc** [comment]
Yes, we only moved the repo over today, this is still a work in progress. We have requested the Docker Hub and will move over when ready.

Thanks for raising the issue, we won't forget now ;-)
